Source/test/test_print_proto.py

```diff
@@ -0,0 +1,15 @@
+from test_harness import *
```

Comment:
:large_orange_diamond: The message on this commit is too long. You can see this when viewing in the PR overview page. Can you consider shortening it?

Reply:
I've shortened the commit message.